components/org.wso2.carbon.identity.captcha/pom.xml

```diff
@@ -87,8 +87,12 @@
             <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.wso2.carbon.identity.event.handler.accountlock</groupId>
```

Comment:
Any particular reason for removing this dependency ? Were you able to make sure that there are no usages of this within captcha ?